src/raftstore/store/worker/apply.rs

```diff
@@ -1176,6 +1187,57 @@ impl ApplyDelegate {
 
         Ok(resp)
     }
+
+    fn handle_ingest_sst(&mut self, req: &Request) -> Result<Response> {
```

Comment:
we may meet the corner case that after handle_ingest_sst but fail to save the apply index, we will retry applying the command again, how do we do?

Reply:
Seems we will not retry, we just panic if failed to write to the engine.

Reply:
I got it, it will still retry on restart after panicked.

Reply:
I think we can create hard links before ingesting, but we need to clean up them after they are applied successfully.

Reply:
This is the same problem as snapshot applying. 

@zhangjinpeng1987 we should support updating sequence number ASAP.